.github/workflows/release_tag.yml

```diff
@@ -0,0 +1,19 @@
+name: Release Tag
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - VERSION
+      - .github/workflows/release_tag.yml
```

Comment:
What if `release_tag.yml` is updated, but the version is not? In such a situation, the tag with the same name is already there. Will this `pxpm/github-tag-action@1.0.1` move the tag to the new commit or simply report an error?

Reply:
> What if release_tag.yml is updated, but the version is not? In such a situation, the tag with the same name is already there. Will this pxpm/github-tag-action@1.0.1 move the tag to the new commit or simply report an error?

After testing, the `pxpm/github-tag-action@1.0.1` will simply report an error and fail the CI.

Reply:
So I guess we should remove `.github/workflows/release_tag.yml` from the list of triggering conditions. This makes the code more robust.

Reply:
> So I guess we should remove .github/workflows/release_tag.yml from the list of triggering conditions. This makes the code more robust.

Yes, I agree. I removed the `.github/workflows/release_tag.yml` from this PR.

Originally I want to include this file so a new tag will be added once this PR is merged, so we can do some other changes with the tag. For example, change the version of aster-frame that OSDK depends on. But this may not be related to this PR. I removed the file now.